[Core] Actor Retries Out of Order Tasks on Restart (#12338)
ray-project/ray: doc/source/fault-tolerance.rst

application is responsible for resubmitting all tasks since the last
checkpoint.

.. note::

>     For :ref:`async actor or threaded actors <async-actors>`, the tasks might

Reviewer: ```suggestion
    For :ref:`async or threaded actors <async-actors>`, the tasks might
```

>     be completed out of order. Upon actor restarts, the system will only retry
>     incomplete tasks in their initial submission order. This prevent unnecessary
>     re-execution of previous completed tasks.

Reviewer 2: ```suggestion
    be completed out of order. Upon actor restart, the system will only retry
    *incomplete* tasks, in their initial submission order. Previously completed
    tasks will not be re-executed.
```
